### ISSUE TITLE
Wallet insecure

### ISSUE THREAD

**Issue by uncle8** [Adolfcoin/adolfcore-zero#26]
As it is now, the wallet doesn't ask for you to retype the password, and it does not give you a deterministic seed to recover it. The password is masked so if you type the wrong password the first time, you will not be able to recover your coins.

In short; making a new wallet should be changed...

**Comment by thunderlark** [comment]
Retyping the password is a trivial change; in fact I might make a patch for that sometime.

The seed can be recovered by typing `seed` at the wallet's command line.

**Comment by uncle8** [comment]
Yes, it's trivial, but it has to be done. The seed should possibly be displayed after a wallet has been created, with instructions on how to use it?

**Comment by thefuhrer** [comment]
> The seed should possibly be displayed after a wallet has been created
This is the current behavior.

Instructions on how to restore from the seed are on the Wiki (which I plan to move to the pinned repository for Adolfcoin).
